code/game/supplyshuttle.dm

```diff
@@ -966,6 +1042,162 @@ var/datum/controller/supply/supply_controller = new()
+		temp += SPAN_WARNING("See you..<BR>")
+		return
+
+	var/rng = rand(1, 100) // Will only sometimes give messages
+	switch(rng)
+		if(1 to 5)
+			temp += "Sometimes I... hear things down 'ere. Crates bein' opened, shufflin', sometimes.. even breathing and chewin'. Even when the ASRS is on maintenance mode.<BR>"
+			temp += "Last month I swear i glimped some shirtless madman runnin' by at the edge of my screen. This place is haunted.<BR>"
+		if(6 to 10)
+			temp += "You know how I said there was a full aisle of autodoc crates? I just found <i>another!</i><BR>"
+			temp += "This one has body scanners, sleepers, WeyMeds.. why the fuck aren't these on the supply list? Why are they here to begin with?<BR>"
+		if(11 to 15)
+			temp += "You know, this place is a real fuckin' massive safety hazard. Nobody does maintenance on this part of the ship.<BR>"
+			temp += "Ever since that CLF operation cost us half the damn cargo hold, nothin' here quite works properly.<BR>"
+			temp += "Mechanical arms dropping crates in random places, from way too high up, knockin' shelves over.. it's fuckin' embarrassin'!<BR>"
+			temp += "I pity the damn' scrappers that'll be trying to salvage something from this junkyard of a ship once it's scuttled.<BR>"
+		if(16 to 20)
+			temp += "I still can't believe the whole ship's fucking supply of HEAP blew up. Some fuckin' moron decided our EXPLOSIVE AMMUNITION should be stored right next to the ship's hull.<BR>"
+			temp += "Even with the explosion concerns aside that's our main damn type of ammunition! What the hell are marines usin' this operation? Softpoint? Jesus.<BR>"
+			temp += "I do see a few scattered HEAP magazines every so often, but I know better than to throw them on the lift. Chances are some wet-behind-the-ears greenhorn is goin' to nab it and blow his fellow marines to shreds.<BR>"
+		if(21 to 25)
+			temp += "Wanna know a secret? I'm the one pushin' all those crates with crap on the ASRS lift.<BR>"
+			temp += "Not because I know you guys need surplus SMG ammunition or whatever. The fuckin' crates are taking up way too much space here. Why do we have HUNDREDS of mortar shells? By god, it's almost like a WW2 historical reenactment in here!<BR>"
+		if(26 to 30)
+			temp += "You know... don't tell anyone, but I actually really like blue-flavored Souto for some reason. Not the diet version, that cyan junk's as nasty as any other flavor, but... there's just somethin' about that blue-y goodness. If you see any, I wouldn't mind havin' them thrown down the elevator.<BR>"
+		if(31 to 35)
+			temp += "If you see any, er.. 'elite' equipment, be sure to throw it down here. I know a few people that'd offer quite the amount of money for a USCM commander's gun, or pet. Even the armor is worth a fortune. Don't kill yourself doin' it, though.<BR>"
```

Comment:
...

Reply:
guh ?

Reply:
harry has said the weird ellpises are fine